libs/utils/version-formatter.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Formats the given version.
+ *
+ * Examples
+ *   '1.2.3' => 'v1.2.3'
+ *   'v1.2.3' => 'v1.2.3'
+ *   'dev-master' => 'dev-master',
+ *   '' => 'unknown'
+ */
+module.exports = (version) => {
+  if (!version) {
```

Comment:
Should we consider the case of number `0`?
Maybe `version = String(version)` at first, make it number proof.

Or add the param type info to comment block, just in case.